frontend/src/views/PrinterControlPage.vue

```diff
@@ -155,7 +162,6 @@ import FailureDetectionWidget from '@src/components/printer-control/FailureDetec
 import TemperatureWidget from '@src/components/printer-control/TemperatureWidget'
 import PrinterControlWidget from '@src/components/printer-control/PrinterControlWidget'
 import ReorderModal from '@src/components/ReorderModal'
-import { getLocalPref } from '@src/lib/pref'
 import SharePrinter from '@src/components/printers/SharePrinter.vue'
```

Comment:
getLocalPref is a simple function that deals with edge cases such as users not having local storage enabled. And using it will keept it consistent with the rest of the code base. Any reason why we shouldn't use getLocalPref here